main/static/style.css

```diff
@@ -13,22 +13,47 @@ body {
 }
 
 footer{
-    padding-top: 30px;
     margin: auto;
-    align: center;
-    /*border-top-color: #5b80b2;*/
-    /*border-top-width: thick;*/
+    text-align: center;
 }
 
-#headerimg{
-    width:128px;
-    height:128px;"
+.match-num{
+    text-align: center;
+}
+
+.match-row-item{
+    background: #7CA0C7;
+    border-radius: 25px;
 }
 
-#footergif{
-    width: 70%;
+.table{
+    width: 100%;
+    display: table;
 }
 
-#content{
-    /*background: blue;*/
-}
\ No newline at end of file
+.table-row{
+    display: table-row;
+    display: block;
+}
+
+.main-table-cell{
+    width: 200px;
+    display: table-cell;
+    text-align: center;
+}
+
+.vs-table-cell{
+    width: 20px;
+    display: table-cell;
+    text-align: center;
+}
+
+.pingimg{
+    width:128px;
+    height:128px;
+}
+
+img:hover {
+    -webkit-transform: scaleX(-1);
+    transform: scaleX(-1);
+}
```

Comment:
Please always add a space between the selector name and the opening curly brace.
Like this: `.pingimg {` (and _not_ this: `.pingimg{` )